### ISSUE TITLE
Test the oceanhackweek image

### ISSUE THREAD

**Issue by ocefpaf** [oceanhackweek/jupyterhub#2]
@salvis2 the image is in the docker hub! Are we ok to test it here?

**Comment by salvis2** [comment]
Yeah! I can change config in the morning, but feel free to do it yourself if you like!

If you want to do it yourself, you can change the lines in `config/staging.yaml` and `config/prod.yaml`. Only push to the staging branch to start, confirm that the new image works on the staging hub at https://staging.ocean.hackweek.io, then make a PR to prod to deploy the change there too.
